### DISCUSSION TITLE
Running on MacOS

### DISCUSSION THREAD

**Discussion by ehotaling** [lukas-blecher/LaTeX-OCR#291] [Q&A]
First off, thank you for taking the time to build this amazing project. I am however having an issue running the model on MacOS. After installing the PyTorch and installing the pix2tex package I am unable to call pix2tex or latexocr from the command line.  I am getting a zsh: command not found: pix2tex error and zsh: command not found: latexocr error for each. What am I doing wrong?

**Comment by lukas-blecher** [comment]
did you activate the correct conda environment in your shell?
How did you install it exactly?

**Reply by ehotaling** [reply]
I did not have a conda environment in my shell, I was just using pip to install the packages. After installing anaconda however I was able to get the issue resolved. Thanks you so much!
